src/server/pfs/server/testing/server_test.go

```diff
@@ -5385,21 +5385,23 @@ func TestPFS(suite *testing.T) {
 		for _, path := range paths {
 			require.NoError(t, env.PachClient.PutFile(commit, path, strings.NewReader(path)))
 		}
+		objC := dockertestenv.NewTestObjClient(ctx, t)
+		bucketURL := objC.BucketURL().String()
 		check := func() {
-			objC := dockertestenv.NewTestObjClient(ctx, t)
```

Comment:
Why was this change needed?

Reply:
I wanted to add a case to test importing a directory and not individual files. My goal was to structure the code to reduce the amount of redunant lines, so I moved some of the steps out of check(). This way I could reset after testing individual file imports and test importing a directory.

Reply:
The check function pattern in the server code is intended to ensure that the same access patterns work before and after finishing a commit. This sort of removes that since it now changes what is being checked against (files vs directory) before and after finishing the commit. I think we should just add the directory case to the check. So, it could be structured like: check files, clear object store, check directories.